client/src/components/TagsMultiselect/StatelessTags.vue

```diff
@@ -124,6 +126,7 @@ function onTagClicked(tag: string) {
             :multiple="true"
             :taggable="true"
             :close-on-select="false"
+            aria-expanded="false"
```

Comment:
avoids mutating the elements attributes directly

```suggestion
            :aria-expanded="editing ? 'true' : 'false'"
```

Reply:
I tried this but just placing editing in there - it didn't work so I assumed Vue would only add these extra options to custom components if they were statically defined which seemed ... inconsistent to me. I will try again with this explicit string casting.

Reply:
Vue seems to for some reason ignore attributes which are set to boolean `false`.
So `:aria-expanded="editing"` only works while `editing` is `true`, otherwise the entire attribute is omitted.
Setting it explicitly to the string `"false"` however works, and achieves the same result from a markup perspective

Reply:
Ahhh - amazing insight - thank you!